OLSF/libra
# [Web monitor] Changes health check from miner to tower

## Motivation

To fix #742

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

(yes)

## Test Plan

1. Start swarm locally, web monitor and check tower is down:
![image](https://user-images.githubusercontent.com/10797037/137822315-36ab572d-39f9-4d8b-b663-3d2d5651c74a.png)

2. Start tower and check health again:
![image](https://user-images.githubusercontent.com/10797037/137822230-1d29d1da-9199-4f8c-a92e-b95042008994.png)
